components/avatar/avatar.html

```diff
@@ -65,18 +65,101 @@ <h2>Avatar - Profile</h2>
+                                </div>
+                                <div>
+                                    <img class="square-avatar size-S " src="/components/images/s4.jpg" alt="square-avatar" srcset="">
+                                </div>
+                            </div>
+
+                        </div>
+                        <div class="component-code">
+                            <code>
+                                <iframe
+                                    src="https://carbon.now.sh/embed?bg=rgba%28187%2C187%2C187%2C1%29&t=solarized+light&wt=sharp&l=auto&width=680&ds=false&dsyoff=100px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=90%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522flex-icons%2520display-container%2522%253E%250A%2509%253Cdiv%2520class%253D%2522%2520%2522%253E%250A%2509%2509%253Cimg%2520class%253D%2522avatar%2520size-XL%2522%2520src%253D%2522%252Fcomponents%252Fimages%252Fp1.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%2509%253C%252Fdiv%253E%250A%2509%253Cdiv%2520%253E%250A%2509%2509%253Cimg%2520class%253D%2522avatar%2520size-L%2520%2522%2520src%253D%2522%252Fcomponents%252Fimages%252Fp1.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%2509%253C%252Fdiv%253E%250A%2509%253Cdiv%2520%253E%250A%2509%2509%253Cimg%2520class%253D%2522avatar%2520size-M%2520%2522%2520src%253D%2522%252Fcomponents%252Fimages%252Fp1.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%2520%2520%2509%253C%252Fdiv%253E%250A%2509%253Cdiv%2520%253E%250A%2509%2509%253Cimg%2520class%253D%2522avatar%2520size-S%2520%2522%2520src%253D%2522%252Fcomponents%252Fimages%252Fp1.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%2509%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                                    style="width: 1000px; height: 300px; border:0; transform: scale(1); overflow:hidden;"
+                                    sandbox="allow-scripts allow-same-origin">
+                                </iframe>
+                            </code>
+                        </div>
+
+                        <div class="component-display">
+                            <h2>Animated - Avatar</h2>
+                            <p class="txt-s padb-1">Could be used in place of display profile based on gender</p>
+                            <div class="flex-icons display-container">
+                                <div class=" ">
+                                    <img class="avatar size-XL" src="/components/images/male.svg" alt="avatar" srcset="">
+                                </div>
+                                <div>
+                                    <img class="avatar size-L " src="/components/images/female.svg" alt="avatar" srcset="">
+                                </div>
+                                <div>
+                                    <img class="avatar size-M " src="/components/images/kidspic.jpg" alt="avatar" srcset="">
```

Comment:
Try implementing lazy loading on these images

Reply:
made the changes thanks